content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,38 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x (except Alpine Linux), 1.1.x or 3.x
+- zlib (`libz.so.1`)
+- libgcc (`libgcc_s.so.1`)
+- C standard library (glibc on most Linux distributions or musl libc on Alpine Linux)
+
+The following table shows the required C standard library version for each supported combination of a Linux distribution, CPU architecture and OpenSSL version:
+
+| Distribution           | CPU architecture | OpenSSL version   | libc version        |
+| ---------------------- | ---------------- | ----------------- | ------------------- |
+| RHEL-like              | x86_64           | 1.0.x, 1.1.x, 3.x | glibc 2.17 or newer |
+| Debian-like            | x86_64           | 1.0.x             | glibc 2.19 or newer |
+| Debian-like            | x86_64           | 1.1.x, 3.x        | glibc 2.24 or newer |
+| RHEL-like, Debian-like | ARM64            | 1.0.x, 1.1.x, 3.x | glibc 2.24 or newer |
+| Alpine Linux           | x86_64, ARM64    | 1.1.x, 3.x        | musl libc 1.2.x     |
```

Comment:
The following two tables show the supported Linux distro' family, OpenSSL versions and C standard libraries for each CPU architecture.
Note that, in case of Prisma not being able to resolve the OpenSSL version on your system (e.g. because it is not installed), it will default to OpenSSL 1.1.x

On `amd64` (`x86_64`) architectures:

| Distro family    | OpenSSL version   | libc version | 
| ---------------- | ----------------- | ------------ |
| Alpine           | 1.1.x, 3.x        | musl 1.2.x+  |
| RHEL             | 1.0.x, 1.1.x, 3.x | glibc 2.17+  |
| Debian or others | 1.0.x             | glibc 2.19+  |
| Debian or others | 1.1.x, 3.x        | glibc 2.24+  |

On `arm64` (`aarch64`) architectures:

| Distro family    | OpenSSL version   | libc version | 
| ---------------- | ----------------- | ------------ |
| Alpine           | 1.1.x, 3.x        | musl 1.2.x+  |
| RHEL             | 1.0.x, 1.1.x, 3.x | glibc 2.24+  |
| Debian or others | 1.0.x, 1.1.x, 3.x | glibc 2.24+  |

```suggestion
The following table shows the required C standard library version for each supported combination of a Linux distribution, CPU architecture and OpenSSL version:

| Distribution           | CPU architecture | OpenSSL version   | libc version        |
| ---------------------- | ---------------- | ----------------- | ------------------- |
| RHEL-like              | x86_64           | 1.0.x, 1.1.x, 3.x | glibc 2.17 or newer |
| Debian-like            | x86_64           | 1.0.x             | glibc 2.19 or newer |
| Debian-like            | x86_64           | 1.1.x, 3.x        | glibc 2.24 or newer |
| RHEL-like, Debian-like | ARM64            | 1.0.x, 1.1.x, 3.x | glibc 2.24 or newer |
| Alpine Linux           | x86_64, ARM64    | 1.1.x, 3.x        | musl libc 1.2.x     |
```